### PR TITLE
Remove redundant get file status call in fuse.open()

### PR DESCRIPTION
**Pull request by LuQQiu**
### What changes are proposed in this pull request?

Remove extra get status call in fuse.open()

### Why are the changes needed?

Reduce one unneeded get metadata call

### Does this PR introduce any user facing changes?

NA

### CODE REVIEW

**Comment by LuQQiu** [comment]
@ssz1997 PTAL, thanks!

**Comment by LuQQiu** [comment]
Error: 3.952 [ERROR] alluxio.fuse.AlluxioJniFuseFileSystemTest.read
Error: 3.952 [ERROR]   Run 1: AlluxioJniFuseFileSystemTest.read:450 Source and dst data should be equal: arrays first differed at element [1]; expected:<1> but was:<0>
Error: 3.953 [ERROR]   Run 2: AlluxioJniFuseFileSystemTest.read:450 Source and dst data should be equal: arrays first differed at element [1]; expected:<1> but was:<0>

File: integration/fuse/src/main/java/alluxio/fuse/file/FuseFileInStream.java [hunk 1]
```diff
@@ -78,7 +79,8 @@ public static FuseFileInStream create(FileSystem fileSystem, FuseReadWriteLockMa
       }
 
       try {
-        FileInStream is = fileSystem.openFile(uri);
+        FileInStream is = fileSystem.openFile(status.get(),
```
Comment (huanghua78) [inline integration/fuse/src/main/java/alluxio/fuse/file/FuseFileInStream.java:82]: I think the real fix is here: the URIStatus is already retrieved from Master.
So we can avoid an extra operation by passing the result to .openFile().

**Review by huanghua78 — APPROVED** [review]
Looks good.

File: integration/fuse/src/main/java/alluxio/fuse/file/FuseFileInStream.java (hunk 1 above)
Comment (LuQQiu) [reply integration/fuse/src/main/java/alluxio/fuse/file/FuseFileInStream.java:82]: Yeah, ideally each client only gets one status  from source of truth (UFS or master) for the whole read process.
The current problem is that without metadata cache enabled, for a single read process we will ask for file metadata for multiple times. We didn't store the file status in client for read/write process.

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**